src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -269,6 +275,38 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
   ngOnInit() {
     if (isPlatformBrowser(this.platformId)) {
       this.stripeService.init();
+
+      // ngx-matomo sets up window._paq internally, and doesn't have
+      // A/B test methods, so we work with the global ourselves.
+      if (environment.matomoAbTest && globalThis.hasOwnProperty('_paq')) {
+        _paq.push(['AbTesting::create', {
+          name: environment.matomoAbTest.name,
+          percentage: 100,
+          includedTargets: [{"attribute":"url","inverted":"0","type":"any","value":""}],
+          excludedTargets: [],
+          startDateTime: environment.matomoAbTest.startDate,
+          endDateTime: environment.matomoAbTest.endDate,
+          variations: [
+            {
+                name: 'original',
+                activate: (_event: any) => {
+                  // No change from the original form.
+                  console.log('Original test variant active!');
+                }
+            },
+            {
+                name: environment.matomoAbTest.variantName,
+                activate: (_event: any) => {
+                  this.alternateCopy = true;
+                  console.log('Copy B test variant active!');
+                }
```

Comment:
```suggestion
                activate: (_event: unknown) => {
                  // No change from the original form.
                }
            },
            {
                name: environment.matomoAbTest.variantName,
                activate: (_event: unknown) => {
                  this.alternateCopy = true;
                }
```

Reply:
As I mentioned on Jira, I think I'd rather keep browser logs on activation, at least for now, to make sure we can diagnose any behaviour we're not expecting.